config/locales/devise.en.yml

```diff
@@ -32,10 +32,16 @@ en:
       success: "Successfully authenticated from %{kind} account."
     passwords:
       no_token: "You can't access this page without coming from a password reset email. If you do come from a password reset email, please make sure you used the full URL provided."
+      nonexisten_email: "Unable to find user with email '%{email}'."
       send_instructions: "You will receive an email with instructions on how to reset your password in a few minutes."
+      send_reset_instructions: "An email has been sent to '%{email}' containing instructions for resetting your password."
       send_paranoid_instructions: "If your email address exists in our database, you will receive a password recovery link at your email address in a few minutes."
       updated: "Your password has been changed successfully. You are now signed in."
+      successfully_updated: "Your password has been successfully updated."
       updated_not_active: "Your password has been changed successfully."
+      empty_parameter: "You must fill out the fields labeled 'Password' and 'Password confirmation'."
+      doesn't_match_with_Password: "Password confirmation doesn't match Password"
```

Comment:
Usually we don't put `'` in key values, and you also put a capital `P`.

I would suggest calling this key: `doesnt_match_with_password`